.github/workflows/deploy-docs.yml

```diff
@@ -1,42 +1,36 @@
-name: Deploy docs
-# This job builds and deploys documenation to github pages.
-# It runs on every push to main with a change in the docs folder.
+# This deploy-docs workflow was created based on instructions from:
+# https://docusaurus.io/docs/deployment
+name: Deploy to GitHub Pages
+
 on:
-  workflow_dispatch:
```

Comment:
no harm in keeping this around just in case we want to manually trigger no?

Reply:
Makes sense. Added it back.